msteams-platform/TOC.yml

```diff
@@ -334,6 +334,12 @@
   items:
   - name: Overview
     href: https://docs.microsoft.com/graph/teams-concept-overview
+  - name: Microsoft Graph Toolkit
+    items:
+    - name: UI Components and Authentication Providers
+      href: https://docs.microsoft.com/graph/toolkit/overview
+    - name: Microsoft Teams and Graph Toolkit
+      href: https://github.com/OfficeDev/M365Bootcamp-TeamsOneProductivityHub
```

Comment:
Can we change this link to point here instead? https://docs.microsoft.com/en-us/graph/toolkit/get-started/build-a-microsoft-teams-tab